rasa/utils/io.py

```diff
@@ -105,13 +106,25 @@ def read_yaml(content: Text) -> Union[List[Any], Dict[Text, Any]]:
      Args:
         content: A text containing yaml content.
     """
+    yaml_parser = _get_yaml_parser()
+    content = _fix_ascii_encoding(content)
+
+    return yaml_parser.load(content) or {}
+
+
+def _get_yaml_parser(typ: Text = "safe", add_version: bool = True) -> YAML:
```

Comment:
optional: I think they just named it `typ` to avoid shadowing Python's `type` :-D Maybe we can use a better name